source/localizable/smart_cart/webhooks.html.md.erb

```diff
@@ -94,7 +97,9 @@ Name           | Type   | Value | Description
 
 Name           | Type   | Value | Description
 :------------: | :----: | :---: | :-----------:
-`state` | Object | `{ "old": "open", "new": "cancelled" }` | The state changed values
+`state` | Object | `{ "old": "open", "new": "cancelled" }` | The state changed values (optional)
+`expires_at` | Object | `{ "old": "YYYY-MM-DDTHH:MM:SS+HH:M", "new": "YYYY-MM-DDTHH:MM:SS+HH:M" }` | The order's current expiration time changes (optional)
```

Comment:
Should we add actual sample values here? (we're in the "values" column, not the type)

Reply:
Added specific values for this one.